lib/fluent/config/dsl.rb

```diff
@@ -22,6 +22,8 @@
 module Fluent
   module Config
     module DSL
+      SYSTEM_PARAM = [:type, :id, :log_level] # Add '@' prefix to avoid deprecation message
```

Comment:
How about `RESERVED_PARAMETERS`?
https://github.com/fluent/fluentd/blob/master/lib/fluent/config/element.rb#L46-L52

Reply:
Any chance to handle `label`, too?

```diff
diff --git a/lib/fluent/config/dsl.rb b/lib/fluent/config/dsl.rb
index c315a3f1..c8a0bb78 100644
--- a/lib/fluent/config/dsl.rb
+++ b/lib/fluent/config/dsl.rb
@@ -22,7 +22,7 @@ require 'fluent/config/element'
 module Fluent
   module Config
     module DSL
-      SYSTEM_PARAM = [:type, :id, :log_level] # Add '@' prefix to avoid depreca
tion message
+      SYSTEM_PARAM = [:type, :id, :log_level, :label] # Add '@' prefix to avoid
 deprecation message
 
       module Parser
         def self.read(path)
```

can handle label parameter with the following configuration:

```nginx
source {
  type :forward
  label "@forward"
}

label('@forward') {
  filter('**') {
    type :stdout
  }

  match('**') {
    type :null
  }
}
```